dspace/config/input-forms.xml

```diff
@@ -107,6 +107,28 @@
        </field>
 
        <field>
+         <dc-schema>elsevier</dc-schema>
+         <dc-element>identifier</dc-element>
+         <dc-qualifier>pii</dc-qualifier>
+         <repeatable>false</repeatable>
+         <label>PII</label>
+         <input-type>onebox</input-type>
+         <hint>Enter the PII for this item.</hint>
+         <required></required>
+       </field>
```

Comment:
This "elsevier.identifier.pii" input field likely should not be enabled by default (as the ScienceDirect deposit steps are also not enabled by default). So, it should be commented out by default (you could add a comment saying that sites which want to enable the ScienceDirect import may wish to enable this field)